src/values/callable_value.rs

```diff
@@ -90,6 +93,18 @@ impl<'ctx> AsValueRef for CallableValue<'ctx> {
 
 impl<'ctx> AnyValue<'ctx> for CallableValue<'ctx> {}
 
+#[llvm_versions(14.0..=latest)]
```

Comment:
I suppose there's no reason to feature gate this impl even if it's only used in 14

Reply:
Removed.